snap7/client.py

```diff
@@ -620,8 +623,8 @@ def wait_as_completion(self, timeout: int) -> int:
 
     def _prepare_as_read_area(self, area: str, size: int) -> Tuple[int, Array]:
         if area not in snap7.types.areas.values():
-            raise NotImplementedError(f"{area} is not implemented in snap7.types")
-        if area == snap7.types.S7AreaTM:
+            raise Snap7Exception(f"{area} is not implemented in snap7.types")
```

Comment:
sorry, i think we should probably just get rid of this snap7exection. I think this is a value error.